src/Pages/hyunchanpark/Login/Login.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router';
+import './Login.scss';
+
+const regObj = {
+  emailRegExp:
+    /[a-zA-Z0-9.-_+!]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]{2,}(?:.[a-zA-Z0-9]{2,3})?/,
+  passwordRegExp: /[a-zA-Z0-9]{5,100}/,
+};
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { userId: '', userPw: '' };
+  }
+
+  validate = (value, regExp) => {
+    const reg = new RegExp(regExp);
+    let isValid = false;
+    if (reg.test(value)) isValid = true;
+    return isValid;
+  };
+
+  validateInputData = (id, pw) => {
+    return (
+      this.validate(id, regObj.emailRegExp) &&
+      this.validate(pw, regObj.passwordRegExp)
+    );
+  };
+
+  handleInput = e => {
+    this.setState(
+      {
+        [e.target.name]: e.target.value,
+      },
+      () => {
+        this.validateInputData(this.state.userId, this.state.userPw);
+      }
+    );
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    if (!this.validateInputData(this.state.userId, this.state.userPw)) return;
+
+    this.props.history.push('/main');
+  };
+
+  render() {
+    const { userId, userPw } = this.state;
+    const isDisabled = this.validateInputData(userId, userPw) ? false : true;
```

Comment:
애초에 boolean을 리턴하는 함수이기 때문에 not 연산자를 활용해서
```suggestion
    const isDisabled = !this.validateInputData(userId, userPw)
```
button tag에서 한번만 사용되는 값이라면 따로 변수선언 할당과정 거칠 필요 없이 버튼 태그 내에 바로 적어줘도 될 것 같습니다!

Reply:
리뷰 반영했습니다.